src/plugins/data/public/query/query_string/language_service/language_service.ts

```diff
@@ -66,7 +66,7 @@ export class LanguageService {
     return Array.from(this.languages.values());
   }
 
-  public getDefaultLanguage(): LanguageConfig {
+  public getDefaultLanguage(): LanguageConfig | undefined {
```

Comment:
+1

Reply:
response type was incorrect. in current state there should never not be a default language. but since it's stored as a map, the linter is saying `DQL` technically can't exist.

probaby need to bucket in having a more thorough default language and what that default language is. or did we want to lean in on perhaps there is no language state.